docker_mcp/core/migration/manager.py

```
@@ -0,0 +1,309 @@
+            # Create archive on source
+            ssh_cmd_source = self._build_ssh_cmd(source_host)
+            archive_path = await self.archive_utils.create_archive(
+                ssh_cmd_source, source_paths, f"{stack_name}_migration"
+            )
+            
+            # Transfer archive to target
+            transfer_result = await transfer_instance.transfer(
+                source_host=source_host,
+                target_host=target_host,
+                source_path=archive_path,
+                target_path=f"/tmp/{stack_name}_migration.tar.gz"
```

Ruff (0.12.2)

Probable insecure usage of temporary file or directory: "/tmp/"

(S108)